pkg/steps/multi_stage/init.go

```diff
@@ -123,15 +134,63 @@ func (s *multiStageTestStep) setupRBAC(ctx context.Context) error {
 			Subjects:   subj,
 		},
 		{
-			ObjectMeta: meta.ObjectMeta{Namespace: s.jobSpec.Namespace(), Name: "test-runner-view-binding", Labels: labels},
+			ObjectMeta: meta.ObjectMeta{Namespace: ns, Name: "test-runner-view-binding", Labels: labels},
 			RoleRef:    rbacapi.RoleRef{Kind: "ClusterRole", Name: "view"},
 			Subjects:   subj,
 		},
 	}
-
+	if s.vpnImage != "" {
+		bindings = append(bindings, rbacapi.RoleBinding{
+			ObjectMeta: meta.ObjectMeta{Namespace: ns, Name: s.name + "-vpn"},
+			RoleRef: rbacapi.RoleRef{
+				Kind: "ClusterRole",
+				// TODO create an official SCC, assign it to the `ci-operator`
+				// ServiceAccount
+				Name: "system:openshift:scc:bbguimaraes-test-scc",
```

Comment:
I'll deploy an official SCC and change this before merging.

Reply:
https://issues.redhat.com/browse/DPTP-2851

Reply:
Small update here: I assigned this test SCC to the production `ci-operator` `ServiceAccount` today and did not observe any issues.  I'll open an `openshift/release` PR with the official deployment then update this value before this PR is merged.

Reply:
Updated to the final name we'll use.

This does not block merging since it is conditionally enabled and no cluster profile currently does.